grr/core/grr_response_core/lib/casing.py

```diff
@@ -0,0 +1,25 @@
+# Lint as: python3
+"""Functions to convert strings between different case styles."""
+import re
+
+
+def SnakeToCamel(snake_str: str) -> str:
+  """Convert a string in snake_case to lowerCamelCase."""
+  if not snake_str:
+    return snake_str
+
+  snake_components = snake_str.split("_")
+  lower_camel_str = (
+    f"{snake_components[0][:1].lower() + snake_components[0][1:]}"
+    f"{''.join([comp.capitalize() for comp in snake_components[1:]])}"
```

Comment:
`f"{str_x + str_y}{str_z}"` looks awkward. Why not just `str_x + str_y + str_z`?

Reply:
The code changed a bit, but I still made the transition to `str + str`. Thank you!